wd_cipher.c

```diff
@@ -317,9 +314,92 @@ void wd_cipher_uninit(void)
 	wd_cipher_setting.priv = NULL;
 	free(priv);
 
+	/* uninit async request pool */
 	wd_uninit_async_request_pool(&wd_cipher_setting.pool);
+
+	/* unset config, sched, driver */
 	wd_clear_sched(&wd_cipher_setting.sched);
 	wd_clear_ctx_config(&wd_cipher_setting.config);
+}
+
+int wd_cipher_init(struct wd_ctx_config *config, struct wd_sched *sched)
+{
+	bool flag;
+	int ret;
+
+	pthread_atfork(NULL, NULL, wd_cipher_clear_status);
+
```

Comment:
any issue if pthread_atfork is called multi times?
if yes, how about put later, since wd_alt_try_init has protection

Reply:
pthread_atfork is used to prevent data exceptions after multi-process fork.
The following wd_alg_try_int is used to prevent repeated initialization of multiple threads in a single process